shared/paths.yaml

```diff
@@ -8,7 +8,8 @@ histoneHMM_path: "/data/manke/repository/scripts/snakemake_workflows/R/x86_64-re
 bedtools_path: "/package/bedtools2-2.25.0/bin/"
 bowtie2_path: "/package/bowtie2-2.2.8/bin/"
 cutadapt_path: "/package/cutadapt-1.9.1/bin/"
-deepTools_path: "/package/deeptools-2.5.1/bin/"
+#deepTools_path: "/package/deeptools-2.5.1/bin/"
+deepTools_path: "module load deeptools; "
```

Comment:
No, using absolute paths is simply the wrong thing to do. Paths change and may differ by machine (fun fact, on maximus /package is a symlink, on some others it's not), but `module load` will always work.

Having said that `module load deeptools/2.5.1` would be better.

Reply:
I am suggesting the use of abspath (atleast till this release 0.8) since 1) it's consistant with other path syntex 3) `module load` is specific to our infrastructure 3) in the next version have to figure out how to replace paths with conda envs anyway

Reply:
A path is just as specific to us as environment modules. Consistency is a non-argument.

Reply:
abspath are still easier to understand. for eg. in the current setup documentation, we write: "replace the paths in paths.yaml with your own paths". Anyway, we have been using module load for snakemake already so that's not much of an issue here.